File: src/main/java/com/tencent/bk/devops/atom/pojo/jfrog/JfrogFile.java
package com.tencent.bk.devops.atom.pojo.jfrog;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;

@Data
public class JfrogFile {
  public String uri;
  public Long size;
  public String lastModified;
  public Boolean folder;

  @JsonProperty(required = false)
  public String sha1;
}

File: src/main/kotlin/com/tencent/bk/devops/plugin/pojo/artifactory/JfrogFile.kt
package com.tencent.bk.devops.plugin.pojo.artifactory

import com.fasterxml.jackson.annotation.JsonProperty
import lombok.Data

@Data
data class JfrogFile(
        val uri: String,
        val size: Long,
        val lastModified: String,
        val folder: Boolean,
        @JsonProperty(required = false)
        val sha1: String = ""
)